Makefile

```diff
@@ -66,13 +66,16 @@ test-func: reinstall
 		-e @$(ROOT_DIR)/tests/auth.yml \
 		$(FUNC_TEST_ARGS) test_all.yml
 
+# FIXME: remove the parameters from here, put them in a file (like we
```

Comment:
I pushed this:
https://github.com/os-migrate/os-migrate/pull/212

To adjust the parameters based on the scenario we will like to run.